op-service/eth/blob_test.go

```diff
@@ -32,10 +33,63 @@ func TestBlobEncodeDecode(t *testing.T) {
 	}
 }
 
+func TestSmallBlobEncoding(t *testing.T) {
+	// the first field element is filled and no data remains
+	data := Data(make([]byte, 128))
+	data[127] = 0xFF
+
```

Comment:
We usually use `"github.com/stretchr/testify/require"` for test assertions.

You can do `require.NoError(b.FromData(data), "failed to encode bytes")`

Reply:
updated to use testify/require

Reply:
thank you!